modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/form21a_controller.rb

```diff
@@ -22,14 +31,33 @@ def submit
 
       attr_reader :parsed_request_body
 
+      def schema
+        # NOTE: This doesn't reject any extra attributes not found in the schema. If
+        # we want that the schema needs to have { "additionalProperties" => false }
```

Comment:
I like the comment but I think it is missing the important point that `additionalProperties` _might_ be the appropriate way to reject extra properties, but not necessarily. This calls out that there are two possibilities for who has the responsibility, the schema itself or the controller (i.e. it is possible to imagine other valid uses of the schema that don't care about additional properties.

Reply:
Another interesting thing is that I don't see any [results](https://github.com/search?q=repo%3Adepartment-of-veterans-affairs%2Fvets-json-schema+additionalProperties+path%3A%2F%5Edist%2F&type=code) when I search for `additionalProperties` in the `dist` directory of `vets-json-schema`.. I wonder what's going on with that.